control/tap-ctl-spawn.c

```diff
@@ -196,5 +232,15 @@ tap_ctl_spawn(void)
 	if (id < 0)
 		EPRINTF("get_id failed, child %d err %d\n", child, errno);
 
+	/* Put the tapdisk in a cgroup slice (best-effort) */
+	if (!slice) {
+#ifdef TAP_CTL_USE_DEFAULT_CGROUP_SLICE
+		/* No option specified; move it to the default slice */
+		tap_ctl_move_to_cgroup(id, "/");
```

Comment:
I know that your intention was that but:
* I'd rather have a message saying "could not do that so I am doing this"
* I am worried that with the new interface coming that sysfs entry won't be there any longer.
This means that we will stop using (silently or just visible in logs when the above point will be addressed) cgroups and we will spot it in specific tests.

So, given using cgroups it is an important thing for us (otherwise you would not have spent
time on this :-D ), we cannot afford the luxury to have it ignored and be unnoticed.